ts/nni_manager/common/shellUtils.ts

```diff
@@ -59,3 +61,13 @@ export function powershellString(str: string): string {
         return singleQuote + str + singleQuote;
     }
 }
+
+export function createScriptFile(path: string, content: string): Promise<void> {
+    // eslint-disable-next-line no-control-regex
+    if (path.endsWith('.ps1') && /^[\x00-\x7F]*$/.test(content)) {
```

Comment:
`/^[\x00-\x7F]*$/` are for ASCII characters. Should we add a `!` for non-ASCII characters?